crates/router/src/connector/powertranz/transformers.rs

```diff
@@ -55,6 +86,12 @@ pub struct PowertranzAddressDetails {
     phone_number: Option<Secret<String>>,
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct RedirectResponsePayload {
+    pub spi_token: String,
```

Comment:
Should this be a secret?

Reply:
This is not related to credential or PII info. we get this info in payment response also just a reference to retrieve 3ds payment